samples/tfm_integration/psa_crypto/Kconfig

```diff
@@ -19,4 +19,20 @@ config PSA_SHELL
 	help
 	  Enabling this option will make the 'psa' shell command available.
 
+choice
+	prompt "Private Key"
+	default PRIVATE_KEY_RANDOM
+
+config PRIVATE_KEY_STATIC
+	bool "Static"
+	help
+	  A static key value will be used for the private key.
```

Comment:
Please mention which key is used.

Reply:
I've addd elliptic curve details.

Reply:
I don't know if @oyvindronningstad  meant the key type (though it's nice to have) ... or where the static key is defined, which is this chunk of code, converting the included .pem key file???

```
	/* secp256r1 private key. */
#if CONFIG_PRIVATE_KEY_STATIC
	/* This value is based on the private key in user.pem,
	 * which can be viewed viw the following command:
	 *
	 *   $ openssl ec -in user.pem -text -noout
	 */
	uint8_t priv_key_data[32] = {
		0x14, 0xbc, 0xb9, 0x53, 0xa4, 0xee, 0xed, 0x50,
		0x09, 0x36, 0x92, 0x07, 0x1d, 0xdb, 0x24, 0x2c,
		0xef, 0xf9, 0x57, 0x92, 0x40, 0x4f, 0x49, 0xaa,
		0xd0, 0x7c, 0x5b, 0x3f, 0x26, 0xa7, 0x80, 0x48
	};
#else
	/* Randomly generate the private key. */
	uint8_t priv_key_data[32] = { 0 };

	psa_generate_random(priv_key_data, sizeof(priv_key_data));
#endif
```

Reply:
Also, I see a typo in the comment, `viw` = `with` (my fault )